### PR TITLE
Add Work and About Us Section

### PR DESCRIPTION
**Pull request by shella12**
#Added Work Section and About Us Section
- Edited HTML
- Edited CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,6 +42,119 @@ <h1 id="header">Ayesha Arshad</h1>
 
         </div>
     </section>
+    <section class="work">
```
Comment (cwaku) [inline index.html:45]: - [x] It is a requirement that you align the main container of the projects with the `display: grid` CSS property. Kindly fix that.

File: style.css [hunk 2]
```diff
@@ -100,6 +100,100 @@
...
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.work .project {
+  width: fit-content;
+  background: #1a2236;
+  color: #fff;
+  padding: 0%;
+}
```
Comment (cwaku) [inline style.css]: - [x] It is a requirement that you align the content of each project with flexbox. Kindly fix that.

File: index.html [hunk 3]
```diff
@@ -39,6 +42,119 @@ <h1 id="header">Ayesha Arshad</h1>
...
+        <div class="project">
+            <img src="images/project1.png" alt="First Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>MySQL</span>
+                <span>PHP</span>
+            </div>
```
Comment (cwaku) [inline index.html]: - [x] I suggest you have the list of the technologies in a `ul` and each of them a `li`. This is part of the core requirements.

File: index.html [hunk 4]
```diff
@@ -39,6 +42,119 @@ <h1 id="header">Ayesha Arshad</h1>
...
+        </div>
+        <div class="project">
+            <img src="images/project3.png" alt="Third Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>Ruby</span>
+                <span>Python</span>
```
Comment (cwaku) [inline index.html]: - [x] On the Figma template, there are only three technologies per project but it happens you have more than that. This is totally alright but it seems to be creating a problem where all of the technologies are not appropriately aligned. Kindly either tweak the CSS to align them as seen in the design or remove the extra altogether.
- [x] While you are at it, kindly confirm that the size of the technologies matches that of the Figma design.

Figma Design | Your Design
-- | --
image | image

File: index.html [hunk 5]
```diff
@@ -39,6 +42,119 @@ <h1 id="header">Ayesha Arshad</h1>
...
+        </div>
+    </section>
+    <section class="about-us">
+        <h1>About</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+            exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        <div class="button"><small>Get my CV </small>
+            <div><i class="fas fa-arrow-down"></i></div>
+        </div>
```
Comment (cwaku) [inline index.html:116]: - [x] The button (e.g., "Get my CV") should be a link `a` or `button` tag and follows the button styling: Active, Hover and Pressed. I suggest you have that fixed.

image

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @shella12 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
- [x] Good overall template design 👍🏾 
- [x] Linters passed 💯 
- [x] Correct use of Github flow 🥇 
- [x] Meaningful PR title 👍🏾 
- [x] Personalised custom README ✔️ 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -100,6 +100,186 @@
...
+
+.project .list-tech {
+  list-style: none;
+  margin: 20px 20px 0 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: flex-start;
+}
```
Comment (wuletawwonte) [inline style.css]: - [x] [OPTIONAL] Please make sure the list of tech stacks is laid out properly. The Figma design is a bit different from your output.

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @shella12 👋🏿👋🏿👋🏿, 

Great work on making the changes requested by a previous reviewer :clap:
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**